src/LeagueToolkit/IO/ReleaseManifestFile/ReleaseManifestFile.fbs

```diff
@@ -0,0 +1,18 @@
+﻿namespace LeagueToolkit.IO.ReleaseManifestFile;
+
+table ReleaseManifestFile
+{
+    ID: long;
```

Comment:
Any reason why this is not a `ulong` anymore ?

Reply:
Not intentional. Fixed.